jenkinsbox.yml

```diff
@@ -10,20 +10,22 @@
   vars:
     project_name: CHANGE_ME
 
-    pear_channels:
-      - pear.drush.org
-    pear_packages:
-      - drush/drush
+    pear_channels: []
+    pear_packages: []
```

Comment:
I think we can remove those variables from `jenkinsbox.yml`, because in `roles/ansible-php-pear/defaults/main.yml` they are also empty.